documents/store-operations/bopis/order-page/gift-card-activation.md

```diff
@@ -2,27 +2,21 @@
 
 The BOPIS App allows store staff to activate gift cards directly while fulfilling orders. This helps in handling gift card items without switching to any other system.
 
-## When Does the Gift Card Activation popup Appear?
+## When Does the Gift Card Activation pop-up Appear?
 
 The activation pop-up opens when a staff member taps on a gift card item from the orders list.
 
 ## Steps to Activate a Gift Card
 
 **Step 1: Enter the Activation Code**  
-The staff member can scan or manually type the serial number printed on the physical gift card. It’s important to make sure the code is correct before moving forward.
+The staff member can scan or manually type the serial number printed on the physical gift card. It must be to make sure the code is correct before moving forward, because it cannot be edited later.
```

Comment:
```suggestion
The staff member can scan or manually type the serial number printed on the physical gift card. It must correct before moving forward, because it cannot be edited later.
```
```suggestion
The staff member can scan or manually type the serial number printed on the physical gift card. It must be to make sure the code is correct before moving forward, because it cannot be edited later.
```